examples/step-30/step-30.cc

```diff
@@ -576,42 +576,42 @@ namespace Step30
                               (neighbor->index() > cell->index() &&
```

Comment:
I believe the strange indentation is a result of a missing parenthesis on line 574.